package.json

```diff
@@ -1,26 +1,25 @@
 {
-  "name": "example",
+  "name": "cosmos-block-example",
   "version": "0.1.0",
-  "repository": "https://github.com/graphprotocol/example-subgraph",
+  "repository": "https://github.com/figment-networks/example-subgraph",
   "license": "MIT",
   "scripts": {
-    "build-contract": "solc contracts/Gravity.sol --abi -o abis --overwrite && solc contracts/Gravity.sol --bin -o bin --overwrite",
     "create": "graph create example --node https://api.thegraph.com/deploy/",
-    "create-local": "graph create example --node http://127.0.0.1:8020",
+    "create-local": "graph create cosmos-example --node http://localhost:8020",
```

Comment:
Also, please move this line below `deploy` (so we have everything `-local` grouped together.